makefiles/pseudomodules.inc.mk

```diff
@@ -298,6 +298,9 @@ PSEUDOMODULES += skald_eddystone
 # define optimized read function of DS18 driver as a pseudo module
 PSEUDOMODULES += ds18_optimized
 
+PSEUDOMODULES += crypto_aes_128
```

Comment:
`crypto_aes` is now `crypto_aes_128`, that should be changed in the `Makefile.dep`s.

Reply:
Also https://github.com/RIOT-OS/RIOT/blob/master/sys/crypto/Kconfig needs to be updated. Some tests are failing because of this. I think something like the following should work:
```difff
diff --git a/sys/crypto/Kconfig b/sys/crypto/Kconfig
index fa070b10c1..ce99ad759a 100644
--- a/sys/crypto/Kconfig
+++ b/sys/crypto/Kconfig
@@ -19,18 +19,24 @@ choice
         The common Crypto block ciphers API has multiple implementations. Choose
         one of the following.
 
-config MODULE_CRYPTO_AES
+config CRYPTO_AES
     bool "AES"
     select MODULE_CRYPTO
 
-config MODULE_CRYPTO_3DES
-    bool "3DES (deprecated)"
-    select MODULE_CRYPTO
-
 endchoice
 
 menu "Crypto AES options"
-depends on MODULE_CRYPTO_AES
+depends on CRYPTO_AES
+
+config MODULE_CRYPTO_AES_128
+    bool "AES-128"
+    default y
+
+config MODULE_CRYPTO_AES_192
+    bool "AES-192"
+
+config MODULE_CRYPTO_AES_256
+    bool "AES-256"
 
 config MODULE_CRYPTO_AES_PRECALCULATED
     bool "Pre-calculate T tables"
```

Reply:
Added the configuration. How do I actually test this ? The configs depend on TEST_KCONFIG. Setting this variable to 1 in my Makefile results in errors.

Reply:
Something like the following should show you which modules are selected when using Kconfig
`env BOARD=samr21-xpro TEST_KCONFIG=1 make info-modules`.

Reply:
It seems that there are still some occurrences of `crypto_aes` in some Makefile.dep files

Reply:
Missed some. Should all be gone now. I also removed the `crypto_%`  pseudomodules now.

Reply:
Thanks